databuilder/databuilder/publisher/elasticsearch_publisher.py

```diff
@@ -5,11 +5,11 @@
 import logging
 from typing import List
 
+from amundsen_common.models.index_map import TABLE_INDEX_MAP
```

Comment:
why did was the table index map moved into common?

Reply:
The mappings were previously duplicated (with some discrepancies) in databuilder `elasticsearch_constants.py` and in common. The one in common was more up-to-date than the one in databuilder. Since the mappings are needed across several repos (databuilder and search), keeping the one in common seemed best. 

To keep backwards compatibility, I have retained the constants in databuilder's `elasticsearch_constants.py` but just pointed them to the definitions in common.

Also there was a todo in the databuilder one to stop using it in favor of common...